File: bronze/1546/python/main.py

if (__name__ == "__main__"):
    num = int(input())
    array = list(map(int, input().split()))[:num]
    sum = 0.0
    max = 0
    for data in array:
        if (data > max):
            max = data
    for i in range(num):
        sum += 1.0 * array[i] / max * 100
    print(sum / num)

File: bronze/1546/c++/main.cpp
#include <iostream>

int main()
{
    int num, max = 0;
    float sum = 0;
    std::cin >> num;
    int array[num];
    for (int i = 0; i < num; i++) {
        std::cin >> array[i];
        if (array[i] > max) {
            max = array[i];
        }
    }
    for (int i = 0; i < num; i++) {
        sum += 1.0 * array[i] / max * 100;
    }
    std::cout << sum / num;
    return 0;
}